pkg/minikube/minikube.go

```diff
@@ -33,60 +31,30 @@ func New(logger micrologger.Logger, builder builder.Builder, tag string) *Miniku
 // BuildImages is a Task that build the required images for both the main
 // project and the e2e containers using the minikube docker environment.
 func (m *Minikube) BuildImages(ctx context.Context) error {
-	m.logger.Log("level", "info", "message", "getting minikube docker environment")
-	env, err := m.getDockerEnv()
```

Comment:
The error was unhandled but this was useful for local testing. I'm using that from time to time.
I'm not sure how it works with localkube, but I think just fine. I think the error should be just handled here.

Reply:
Do you mean we should keep the code I removed? Because this does not work at all. CI fails then. I removed it and now it works. What shall I do?

Reply:
For the record. We thought, we talked, no one is sure. Let's delete it and if needed implement something we can explain.